skycultures/ticuna/description.en.utf8

```diff
@@ -0,0 +1,22 @@
+<p align="justify">The Ticuna live in riverside villages along the Amazonas-Solimões tributary igarapés (or headwaters). Their daily life revolves around the cultivation cycles of different agricultural goods, most particularly manioc, which is used to produce the flour sold by Ticuna in the markets of the main cities of the region. They are also fishers, hunters and gatherers of the forest products. The women make hammocks, bags and other objects with plant fibers and the men paint on tururi fiber and fabricate arrows, paddles and canoes.</p>
+<p align="justify">The legends say that the mythical twin brothers Yoi and Ipi created the Ticuna. Ipi’s flesh was converted into fish and Yoi caught these fish in the mythical Eware igarapé. The fishes transformed themselves into Ticuna people when brought upon land. Yoi taught them to work and celebrate and he gave them clan names associated with birds and land animals to organize the Ticuna society.</p>
+<h3 align="center"> The Astronomic Correlations</h3>
+<p align="justify">The astronomic correlations between Western and Ticuna constellations were identified by the Ticuna in November 2002 with the help of the astronomer Fernando Vieira (Planetarium Foundation of Rio de Janeiro City) during a study in the Science Center and Planetarium of the Universidade do Pará (CCPP) in the city of Belém   (Brazil), also using the Starry Night System. We are integrating in this investigation a Physics Professor at Pontifícia Universidade Católica de São Paulo (PUC-SP), and Astronomy in Culture researcher Walmir Thomazi Cardoso, who made some reinterpretations in a collaborative work with physics undergraduate student and Stellarium developer collaborator Youssif Ghanthous Filho, adapting and anchoring original drawings to the Stellarium. Nevertheless, every sky interpretation is contextual and circumstantial. Current collaborative investigations are looking for new astronomical observation together with the Ticuna, which will improve the initial findings.</p>
+<p align="justify">Correlating the Ticuna constellations with those of Western astronomy<cite id="cit2a"><a href="#cit2"><B>²</B></a></cite> , the form of Baweta (the Turtle Collective) is associated with the stars of the asterisms of the Pleiades and of Perseus. Its eyes are the feet of Perseus (Omicron Persei and Atik). The head of the V form that Western astronomy relates to Taurus is called Coyatchicüra (Cayman’s Jaw). In a Ticuna drawing, the Cayman’s left paw corresponds to the star Euratis and the right rear paw to ε (Epsilon), η (Eta) ζ (zeta). The tail is in ο (Omicrom) and ζ (Zeta) in Perseus. In turn, Wücütcha’s Leg is related to the stars that groups as Orion. The toes are Orion’s Belt and the bend of the legs are α (Alfa), γ (Gama),β (Beta), ε (Epsilon).</p>
```

Comment:
Star "Euratis": I cannot find this name. Please add Bayer letter or Flamsteed number. 
Omicrom-->Omicron
Gama --> Gamma

Reply:
Please remove align="justify" and align="center" parts